loader/loader.go

```diff
@@ -582,7 +582,9 @@ func SplitPrefix(path string) ([]string, string) {
 	}
 	parts := strings.SplitN(path, ":", 2)
 	if len(parts) == 2 && len(parts[0]) > 0 {
-		return strings.Split(parts[0], "."), parts[1]
+		if strings.HasPrefix(parts[1], "file:///") || len(parts[0]) > 1 {
```

Comment:
Why do we put significance on the path containing `file:///` or the document prefix being more than one chars long for us to return those parts rather than the given `path`?

Reply:
To add some context: I believe the prefix part is the prefix for which the contents of that file will be found under the `data` document during evaluation.

E.g.: given the following data file `data.json`:

```json
{
  "foo": 42
}
```

and we run: 

```cmd
$ opa eval -fpretty -d file:///path/to/my/file/data.json 'data = x'
```

we expect the output: 
```cmd
+------------+
|     x      |
+------------+
| {"foo":42} |
+------------+
```

But if we add the `a:` prefix to the file path, we expect this prefix to be added to the data document:

```cmd
opa eval -fpretty -d a:file:///path/to/my/file/data.json 'data = x'
```

```cmd
+------------------+
|        x         |
+------------------+
| {"a":{"foo":42}} |
+------------------+
```